### PR TITLE
Change meetings component filter to show all meetings instead of upcomings

### PR DESCRIPTION
**Pull request by davidbeig**
This PR changes the default filter for meetings inside the component view so that all meetings are shown, instead of just the upcoming ones.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require "rails-controller-testing"
```
Comment (verarojman) [inline spec/spec_helper.rb:19]: You can probably remove that now, right?

File: config/initializers/meetings_controller_override.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      {
+        search_text: "",
+        date: %w(all),
+        activity: "all",
+        scope_id: default_filter_scope_params,
+        category_id: default_filter_category_params,
+        origin: default_filter_origin_params,
+        type: default_filter_type_params
+      }
+   end
```
Comment (verarojman) [inline config/initializers/meetings_controller_override.rb:13]: Rubocop complains about this 

```suggestion
    end
```

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -15,6 +15,9 @@
...
+
```
Comment (verarojman) [inline spec/spec_helper.rb:20]: You can probably remove that now, right?

**Review by verarojman — CHANGES_REQUESTED** [review]
Hi! Looks good to me, just run rubocop on `meetings_controller_override.rb` and undo changes to `spec_helper.rb` and it will be 🍒🍒🍒 🏆 👨🏾‍🍳

File: spec/spec_helper.rb (hunk 3 above)
Comment (davidbeig) [reply spec/spec_helper.rb:20]: True

**Comment by davidbeig** [comment]
> Hi! Looks good to me, just run rubocop on `meetings_controller_override.rb` and undo changes to `spec_helper.rb` and it will be cherriescherriescherries trophy 👨🏾‍🍳

Done! In the last commit you can find the fixes :)

**Review by verarojman — APPROVED** [review]

**Pull request merged**